plugins/package-managers/node/src/main/kotlin/npm/Npm.kt

```diff
@@ -145,10 +145,11 @@ class Npm(
     override fun createPackageManagerResult(projectResults: Map<File, List<ProjectAnalyzerResult>>) =
         PackageManagerResult(projectResults, graphBuilder.build(), graphBuilder.packages())
 
-    private fun listModules(workingDir: File): ModuleInfo {
-        val json = NpmCommand.run(workingDir, "list", "--depth", "Infinity", "--json", "--long").requireSuccess().stdout
+    private fun listModules(workingDir: File, issues: MutableList<Issue>): ModuleInfo {
+        val processCapture = NpmCommand.run(workingDir, "list", "--depth", "Infinity", "--json", "--long")
```

Comment:
I propose a variable name that tells a bit more about the command being run, like `listProcess`.